devtools/src/main/scala/kalix/devtools/impl/DevModeSettings.scala

```diff
@@ -16,17 +16,34 @@
 
 package kalix.devtools.impl
 
-import scala.jdk.CollectionConverters._
+import com.typesafe.config.{ Config, ConfigFactory }
 
-import com.typesafe.config.Config
-import com.typesafe.config.ConfigFactory
+import scala.jdk.CollectionConverters._
 
 object DevModeSettings {
 
   val portMappingsKeyPrefix = "kalix.dev-mode.service-port-mappings"
+  val tracingConfigEnabled = "kalix.proxy.telemetry.tracing.enabled"
+  val tracingConfigEndpoint = "kalix.proxy.telemetry.tracing.collector-endpoint"
 
   def fromConfig(config: Config): DevModeSettings = {
+    val devModeSettingsAddedPorts = addPortsFromConfig(config)
+    addTracingConf(config, devModeSettingsAddedPorts)
+  }
 
+  private def addTracingConf(config: Config, current: DevModeSettings): DevModeSettings = {
+    val tracingPort: Option[Int] =
+      if (config.hasPath(DevModeSettings.tracingConfigEndpoint)) {
```

Comment:
Right. I see now. 

Question. Wouldn't it be better to pass to the `WebClientProviderImpl` the config that the `KalixRunner` already is aware of after using `addDevModeConfig` than extract the settings again in the `WebClientProviderImpl.devModeSettings`? 

I mean when creating the `webClientProvider` in  `KalixSpringApplication.webClientProvider` don't we have the host and the ports already because we have run `KalixRunner.loadPreparedConfig()`?

Not even sure I'm right, though.

Reply:
removed.